app/models.py

```diff
@@ -247,22 +247,22 @@ def generate_fake(count=1000, user_count=1):
         from sqlalchemy.sql.expression import func
 
         seed()
-        
+
         user_query = User.query.all()
         u_ids = [i.id for i in user_query]
-        u_ids = sample(u_ids, user_count)        
+        u_ids = sample(u_ids, user_count)
 
         codes_query = Codes.query.filter(Codes.end_date == None).all()
         project_codes_query = ProjectCodes.query.filter(Codes.end_date == None).all()
-        
+
         c_ids = [i.code_id for i in codes_query]
         pc_ids = [i.project_code_id for i in project_codes_query]
-        
+
         for i in range(int(count/user_count)):
 
-            raw_query = Priority.query.first()
+            raw_query = Raw.query.first()
```

Comment:
This does need to come from the Priority view, not the Raw table, as this function is designed to classify in a manner most like real humans (which would be picking the first row from the priority view).
When deploying, a Priority table will be created, and then needs to be dropped and replaced with a Priority View by running the `sql/views/priority.sql`. There probably is a better way of doing that...